pkg/doc.txt

```diff
@@ -1,15 +1,44 @@
-/*
- * Copyright (C) 2015 INRIA
- *
- * This file is subject to the terms and conditions of the GNU Lesser
- * General Public License v2.1. See the file LICENSE in the top level
- * directory for more details.
- */
-
 /**
- * @defgroup    pkg Packages
- * @brief       RIOT supports the integration of external libraries. These
- *              libraries can be used as BSD-like packages, i.e. they are
- *              downloaded automatically at compile-time (if needed) and linked
- *              against RIOT.
+ * @defgroup pkg   Packages
+ * @{
+ * @brief          External libraries and applications
+ *
+ * Using packages
+ * ==============
+ * To add a package to the list of compiled modules you have to add it to the
+ * `USEPKG` macro in your application's Makefile. If the package provides
+ * header files you might need to update the `INCLUDE` macro, too:
+ *
+ * ~~~~~~~~ {.mk}
+ * USEPKG += <pkg_name>
+ * INCLUDE += $(RIOTBASE)/pkg/<pkg_name>/...
```

Comment:
I think that's overkill. Also the `pkg` directory might be the include directory.